state/execution.go

```diff
@@ -466,3 +464,19 @@ func ExecCommitBlock(
 	// ResponseCommit has no error or log, just data
 	return res.Data, nil
 }
+
+// Make pretty string for validatorUpdates logging
+func makeValidatorUpdatesLogString(validatorUpdates []abci.ValidatorUpdate) string {
+	validators, err := types.PB2TM.ValidatorUpdates(validatorUpdates)
```

Comment:
We should have `updateValidators` return the `updates` so we don't have to call this again